src/app/public/modules/popover/popover-adapter.service.spec.ts

```diff
@@ -447,7 +453,7 @@ describe('SkyPopoverAdapterService', () => {
         const popover = new ElementRef(createElementRefDefinition(-20, 0, 100, 100));
         const parentElement = createElementRefDefinition(0, 0, 1200, 800);
 
-        popover.nativeElement.parentNode = parentElement;
+        (popover.nativeElement as any).parentNode = parentElement;
```

Comment:
Why are we doing all these as `any`? Wouldn't it make more sense to use `HTMLElement` or whatever is appropriate?

Reply:
We're creating an instance of `ElementRef` in the unit test; however, the `parentNode` property is readonly, which throws a compilation error (because we need to overwrite the value for the sake of the test). Casting it to `any` creates the same test result, but forces TypeScript to ignore the warning.

Reply:
Ok